roles/openshift_node/tasks/node_system_container.yml

```diff
@@ -16,19 +16,21 @@
   register: update_result
   changed_when: "'Extracting' in update_result.stdout"
   when:
-  - (openshift.common.version is defined) and (openshift.common.version == openshift_version) and ("node" in result.stdout) | bool
+  - l_is_same_version
+  - ("node" in result.stdout) | bool
```

Comment:
I'd suggest using blocks for the following conditions if possible:
- `'node' in result.stdout`
- `not l_is_same version`
- `openvswitch' in result.stdout`

I would also suggest using `check_master_system_container` rather than `result` for the registered variable to avoid confusion.

Should there also be separate variables for `l_node_is_same_version` and `l_openvswitch_is_same_version` ? It looks like openvswitch containers are being managed by the openshift version and not the openvswitch version.